tests/jobs/test_local_task_job.py

```diff
@@ -902,7 +902,9 @@ def test_fast_follow(
             ti_by_task_id = {}
             with create_session() as session:
                 for task_id in init_state:
-                    ti = TaskInstance(dag.get_task(task_id), run_id=dag_run.run_id, state=init_state[task_id])
+                    ti = TaskInstance(dag.get_task(task_id), run_id=dag_run.run_id)
+                    ti.refresh_from_db()
```

Comment:
Wondering if the `refresh_from_db` could be moved to the `TaskInstance` constructor somehow as an optional operation?